### ISSUE TITLE
CA1825: False positive on .NET 11 Preview1

### ISSUE THREAD

**Issue by bkoelman** [dotnet/roslyn#82484]
**Version Used**: 
.NET 11 Preview 1 with VS Insiders 2026 [11513.90]

This is a regression. The problem does not occur using stable .NET 10 and stable VS 2026.

**Steps to Reproduce**:
Using xUnit, a suggestion is reported on the following lines:
```c#
public sealed class ModelStateValidationTests : IClassFixture<OpenApiTestContext>
{
    public static readonly TheoryData<string> SchemaNames =
    [ /* SUGGESTION HERE */
        "attributesInCreateTypeContainerRequest",
        "attributesInUpdateTypeContainerRequest",
        "attributesInTypeContainerResponse"
    ];
}
```

Applying the fix changes nothing. After applying it, the fix disappears from the context menu.

**Diagnostic Id**:
CA1825: Avoid unnecessary zero-length array allocations.  Use Array.Empty<string>() instead.

**Expected Behavior**:
No suggestion.

**Actual Behavior**:
A suggestion is offered. We've increased the severity to Warning in Release builds. Combined with warnaserror, this breaks the build.
